contollers/list.controller.js

```diff
@@ -0,0 +1,63 @@
+const List = require("../models/list.model");
+
+class ListController {
+  static async index(req, res) {
+    try {
+      res.send(await List.find({}));
+    } catch (err) {
+      console.log(err);
+      res.sendStatus(500);
+    }
+  }
+
+  static async create(req, res) {
+    try {
+      let list = await List.create({
+        title: req.body.title,
+        description: req.body.description,
+        userId: req.user.id,
+      });
+
+      res.send(list);
+    } catch (err) {
+      console.log(err);
+      res.sendStatus(500);
+    }
+  }
+
+  static async show(req, res) {
+    try {
+      const result = await List.findById(req.params.id);
+      if (result) res.send(result);
```

Comment:
just send result even if its undefined, its bad practice to change the type of the response (List or String)